samples/nrf_desktop/src/modules/hid_state.c

```diff
@@ -526,15 +526,23 @@ static void send_report_keyboard(void)
 		struct hid_keyboard_event *event = new_hid_keyboard_event();
 
 		event->subscriber = state.selected->id;
+		event->modifier_bm = 0;
 
 		size_t cnt = 0;
 		for (size_t i = 0;
 		     (i < max) && (cnt < ARRAY_SIZE(event->keys));
-		     i++, cnt++) {
+		     i++) {
 			struct item item = rd->items.item[max - i - 1];
 
 			if (item.value) {
-				event->keys[cnt] = item.usage_id;
+				if (item.usage_id < 0x66) {
```

Comment:
What is 0x66, 0xE0, 0xE7? Can you name them or at least comment?

Reply:
Fixed